astroquery/mast/tests/test_mast_remote.py

```diff
@@ -290,21 +304,29 @@ def test_observations_download_products(self, tmpdir):
         assert os.path.isfile(result2['Local Path'][0])
         assert len(result2) == 1
 
-    def test_observations_download_products_no_duplicates(self, tmpdir, caplog):
+    def test_observations_download_products_flat(self, tmp_path, msa_product_table):
+
+        # Get a product list with 6 duplicate JWST MSA config files
+        products = msa_product_table
 
-        # Pull products for a JWST NIRSpec MSA observation with 6 known
-        # duplicates of the MSA configuration file, propID=2736
-        products = mast.Observations.get_product_list("87602009")
+        # Download with flat=True
+        manifest = mast.Observations.download_products(products, flat=True,
+                                                       download_dir=tmp_path)
 
-        # Filter out everything but the MSA config file
-        mask = np.char.find(products["dataURI"], "_msa.fits") != -1
-        products = products[mask]
+        downloaded_path = manifest["Local Path"][0]
+        filename = os.path.basename(downloaded_path)
+        assert downloaded_path == (tmp_path / filename).as_posix()
```

Comment:
Wouldn't this be equivalent?
```suggestion
        assert Path(manifest["Local Path"][0]).parent == tmp_path
```

Reply:
Yes!  Much better.  Thanks.  I'm a little rusty with `pathlib`.  Fixed (with a `from pathlib import Path` at the top).